config/zones/BE.yaml

```
@@ -46,10 +46,10 @@ emissionFactors:
         value: 144.20803925394276
     biomass:
       datetime: '2021-01-01'
-      source: EU-ETS, ENTSO-E 2021
+      source: BEIS 2021
       value: 0.0
     coal:
-      datetime: '2020-01-01'
+      datetime: '2021-01-01'
       source: EU-ETS, ENTSO-E 2021
       value: 1057.262766
```

Not feedback related to this pr in particular but shouldn't it be:
```suggestion
    - datetime: '2014-01-01'
      source: IPCC 2014
      value: 820
    - datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021
      value: 1057.262766
```
It would increase the transparency of the sources (even if these are the defaults) and would make it more clear when looking directly at the files.

Just an idea though so don't let it be a blocker for this PR.

---

Thanks for your comment Viktor. We'll see how we incorporate factors for previous years. We want to compute them for EU as the data is available